packages/miniflare/test/index.spec.ts

```diff
@@ -1327,7 +1327,7 @@ test("Miniflare: modules in sub-directories", async (t) => {
 	t.is(await res.text(), "123");
 });
 
-test("Miniflare: python modules", async (t) => {
+test.skip("Miniflare: python modules", async (t) => {
```

Comment:
What?

Reply:
oops, thanks a lot for noticing it, I mistakenly committed this 🙇

Reply:
removed, thanks a lot for spotting it! 🙏